src/containers/HomePage.js

```diff
@@ -83,6 +86,20 @@ class HomePage extends PureComponent {
     }
   }
 
+  showOrHideTargetSelected = (key) => {
+    const oldSelectedElements = document.getElementsByClassName('marker-point__selected');
```

Comment:
Normally, when on React, we avoid using document's API this way. It's an old way to handle changes in the dom.

You can make the Target component have a `selected` boolean prop, depending on which you would add the `__selected` class or not. Then you need to pass that variable on to the Target from redux. You have the selected target's id on a reducer, right?